rts/sra/pom.xml

```diff
@@ -11,52 +11,76 @@
     <artifactId>sra-parent</artifactId>
     <name>SRA PoC</name>
     <description>SRA PoC</description>
-    <packaging>war</packaging>
+    <packaging>jar</packaging>
 
     <properties>
         <failOnMissingWebXml>false</failOnMissingWebXml>
     </properties>
+     <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-bom</artifactId>
+                <version>${version.quarkus}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
 
     <build>
-        <finalName>lra-test</finalName>
-        <plugins>
+        <finalName>sra-test</finalName>
+         <plugins>
```

Comment:
When we use indentation try to be consistent. Normally we indent by 4 spaces but this file is using a random mix of 1, 2 and 4 spaces.